### PR TITLE
new message data for authentication message.

### PR DESCRIPTION
**Pull request by Nazianzenov**
* message data and test
* different json for invalid verification testing

### CODE REVIEW

**Review by MariemBaccari — APPROVED** [review]
The go code looks good to me. I didn't check very deeply the json files but I think that's up to the popcha reviewer !

File: be1-go/message/test/messagedata/authenticate_user_test.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+	var authUser messagedata.AuthenticateUser
+
+	// action and object are constant
+	object, action := "popcha", "authenticate"
+
+	// generic method, takes a boolean argument, to assert whether the given json should be valid or not
+	getTestValid := func(file string, shouldBeValid bool) func(*testing.T) {
+		return func(t *testing.T) {
+			// read the bad example file
```
Comment (Anghul) [inline be1-go/message/test/messagedata/authenticate_user_test.go:31]: This function works with both good and bad examples, this comment is therefore wrong

**Review by Anghul — COMMENTED** [review]
LGTM, just a small nitpick about a comment. I will aprove the PR once the json schmas are approved by someone from popcha

**Review by dayan9265 — APPROVED** [review]
The JSON files look good to me

File: be1-go/message/test/messagedata/authenticate_user_test.go (hunk 1 above)
Comment (Nazianzenov) [reply be1-go/message/test/messagedata/authenticate_user_test.go:31]: Indeed, i will change that!

**Review by Anghul — APPROVED** [review]
LGTM

**Pull request merged**